tests/foreman/cli/test_http_proxy.py

```diff
@@ -175,3 +175,24 @@ def test_positive_set_content_default_http_proxy(block_fake_repo_access):
     assert rpm_repo.read().content_counts['rpm'] == 0
     product.sync()
     assert rpm_repo.read().content_counts['rpm'] >= 1
+
+
+@pytest.mark.stubbed
+@pytest.mark.tier3
+def test_positive_environment_variable_unset_set():
+    """Verify that satellite installer unsets and then sets back the environment variables
+
+    :id: 596d753b-660b-49cb-b663-ff3cec439564
+
+    :BZ: 1886040
+
```

Comment:
```suggestion

:customerscenario: true
```
Both bugzilla's have customer cases attached to them.

Reply:
Also add necessary `:CaseImportance: ` here